src/utils/classifier_report.py

```diff
@@ -231,6 +258,122 @@ def create_html_page(self, data, csv_path, stats=False):
+                        # updates review export button
+                        a('''
+                          let updateReviewButton = () => {
+                            let reviewItems = localStorage.getItem("reviewItems");
+                            let len = (reviewItems == null) ? 0 : JSON.parse(reviewItems).length;
+                            $("#reviewExportButton").text(`Export ${len} reviewed items`)
+                            if (len == 0) {
+                                    $("#reviewExportButton").prop("disabled",true);
+                                    $("#buttonsBottom").css("visibility", "hidden");
+
+                            } else {
+                                $("#reviewExportButton").prop("disabled",false);
+                                $("#buttonsBottom").css("visibility", "visible");
+
+                            }
+                          }
+                           ''')
+                        # loads review choices from local storage
+                        a('''
+                          let loadReviewChoices = () => {
+                          if (!localStorage.getItem('reviewItems')) {
+                            localStorage.setItem("reviewItems", "[]");  
+                          } else {
+                            let reviewedItems = JSON.parse(localStorage.getItem("reviewItems"))
+                            $('.reviewButton').removeClass('active');
+                            reviewedItems.forEach(e => {
+                                e = JSON.parse(e);
+                               if ($(`#${e['id']}`).hasClass('active') === false) {
+                                  $(`#${e['id']}`).toggleClass('active');
+                                }
+                            })
+                          }
+                        }
+                           ''')
+                        # initialize localStorage object
+                        a('''
+                        if (localStorage.length == 0) {
+                            localStorage.setItem("reviewItems", "[]");  
+                            }
+                        updateReviewButton();
+                           ''')
+                        # runs update functions
+                        a('''
+                          loadReviewChoices();
+                          updateReviewButton();
+                          ''')
+                        # functions to add or remove reviewed items to local storage
+                        a('''
+                            let removeReviewItem = (id) => {
+                                let reviewItems = JSON.parse(localStorage.getItem("reviewItems"));
+                                reviewItems = $.grep(reviewItems, (x) => JSON.parse(x).id != id);
+                                localStorage.setItem("reviewItems", JSON.stringify(reviewItems));
+                            };
+
+                            let addReviewItem = (e) => {

```

Comment:
I'd suggest having removeReviewItem and addReviewItem both take the same input and leave it up to the caller figure out how to get the correct input. So probably pass in `id` in both cases rather than `e` for one of them.